fuzzers/kirenenko/buildScript/lcms-2017-03-21/cms_transform_fuzzer.c

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdint.h>
```

Comment:
For the fuzz target source files as well, can we avoid the redundant copies?

It looks like the changes are turning the c++ files into c. Please feel free to turn the original .cc files into .c file and use the C compiler everywhere, and that way we don't need these very similar copies. Would that work?

Reply:
Got it. In addition to turning the filename, we also convert some C++ syntax to its C equivalent. (e.g. new -> malloc). I will try to fix it in the script.

Reply:
> Got it. In addition to turning the filename, we also convert some C++ syntax to its C equivalent. (e.g. new -> malloc). I will try to fix it in the script.

Yes, please feel free to make these changes in the fuzz target source files and turn the C++ bits to C in the original files. It's important that we build the same fuzz target source for all fuzzers.

Reply:
> It looks like the changes are turning the c++ files into c. Please feel free to turn the original .cc files into .c file and use the C compiler everywhere, and that way we don't need these very similar copies. Would that work?

In OSS-Fuzz we advocate using clang++ to link things. I'm not 100% sure why and I don't know if linking with `clang` will fail. but I'm quite sure some of these projects are written in C++. So what's the point of changing it here?

Reply:
> > It looks like the changes are turning the c++ files into c. Please feel free to turn the original .cc files into .c file and use the C compiler everywhere, and that way we don't need these very similar copies. Would that work?
> 
> In OSS-Fuzz we advocate using clang++ to link things. I'm not 100% sure why and I don't know if linking with `clang` will fail. but I'm quite sure some of these projects are written in C++. So what's the point of changing it here?

Ah, I bet kirienko only supports pure C projects right?

Reply:
Yes, currently Kirenenko only supports C projects.